server/routes/filterTermValues.ts

```diff
@@ -65,7 +65,8 @@ async function getFilters(query, ds, genome, res) {
 		const samplesPerFilter = await getSamplesPerFilter(query, ds)
 		const filtersData = await getData(
 			{
-				terms: query.terms
+				terms: query.terms,
```

Comment:
Here should be the same as in the scores, we dont filter as we show aggregated data.